inference-engine/src/multi_device/multi_device_exec_network.cpp

```diff
@@ -137,6 +137,42 @@ void MultiDeviceExecutableNetwork::GenerateWorkers(const std::string& device, co
     }
 }
 
+void MultiDeviceExecutableNetwork::IncreaseWorkers(AutoLoadContext& loadcontext, InferenceEngine::SoIInferRequestInternal& request_to_share) {
+    auto devicename = loadcontext.deviceInfo.deviceName;
+     auto& executableNetwork =  loadcontext.executableNetwork;
+    auto& idleWorkerRequests = _idleWorkerRequests[devicename];
+    //_inferPipelineTasksDeviceSpecific[devicename] = std::unique_ptr<ThreadSafeQueue<Task>>(new ThreadSafeQueue<Task>);
+    auto* idleWorkerRequestsPtr = &(idleWorkerRequests);
+    {
+        std::lock_guard<std::mutex> lock(_idleWorkerMutex);
+        idleWorkerRequests.set_capacity(idleWorkerRequests.capacity() + 1);
+    }
+    auto* workerRequestPtr =  new WorkerInferRequest;
```

Comment:
need destroy it in some place, if not , memory leak

Reply:
yes, will try to figure out a way to handle this